lib/coinbase/address.rb

```diff
@@ -119,21 +129,34 @@ def to_s
 
     private
 
-    # Normalizes the amount of ETH to send based on the asset ID.
+    # Normalizes the amount of Wei to send based on the asset ID.
     # @param amount [Integer, Float, BigDecimal] The amount to normalize
     # @param asset_id [Symbol] The ID of the Asset being transferred
-    # @return [BigDecimal] The normalized amount in units of ETH
-    def normalize_eth_amount(amount, asset_id)
+    # @return [BigDecimal] The normalized amount in units of Wei
+    def normalize_wei_amount(amount, asset_id)
+      big_amount = BigDecimal(amount.to_s)
+
       case asset_id
       when :eth
-        amount.is_a?(BigDecimal) ? amount : BigDecimal(amount.to_s)
+        big_amount * Coinbase::WEI_PER_ETHER
       when :gwei
-        BigDecimal(amount / Coinbase::GWEI_PER_ETHER)
+        big_amount * Coinbase::GWEI_PER_ETHER * Coinbase::WEI_PER_GWEI
```

Comment:
Shouldn't this be big_amount * Coinbase::GWEI_PER_ETHER? Why are we multiplying again by WEI_PER_GWEI?